win32/devdoc/job_object_helper.md

```diff
@@ -0,0 +1,46 @@
+# job_object_help requirements
+
+`job_object_help` is a module that wraps the Windows JobObject API, primarily for the purpose of limiting the amount of memory and CPU the current process can consume.
+
+## Exposed API
+```c
+    MOCKABLE_FUNCTION(, int, job_object_helper_limit_resources, uint32_t, percent_physical_memory, uint32_t, percent_cpu);
+```
+
+## job_object_helper_limit_resources
+```c
+    MOCKABLE_FUNCTION(, int, job_object_helper_limit_resources, uint32_t, percent_physical_memory, uint32_t, percent_cpu);
+```
+
+`job_object_helper_limit_resources` is an API that uses the job object API to limit the amount of physical memory and CPU that the current process can consume.
+
+**SRS_JOB_OBJECT_HELPER_18_001: [** `percent_physical_memory` is allowed to be `0`, which means "do not limit physical memory usage". **]**
+
+**SRS_JOB_OBJECT_HELPER_18_002: [** `percent_cpu` is allowed to be `0`, which means "do not limit CPU usage". **]**

```

Comment:
We usually don't spec which values are valid. 

How would you test these two specs exclusively without hitting _003?

Reply:
removed.  003 only comes into play if they are both zero.

Reply:
rewritten to simplify and remove redundancy.